packages/esm-patient-immunizations-app/src/immunizations/immunizations-form.component.tsx

```diff
@@ -158,157 +124,109 @@ const ImmunizationsForm: React.FC<ImmunizationsFormProps> = ({
-              )}
-              <div className={styles.immunizationsInputContainer}>
-                <label htmlFor="vaccinationDate">{t('vaccinationDate', 'Vaccination Date')}</label>
-                <div className="omrs-datepicker">
-                  <input
-                    type="date"
-                    id="vaccinationDate"
-                    name="vaccinationDate"
-                    max={today}
-                    required
-                    defaultValue={formState.vaccinationDate}
-                    onChange={(evt) => updateSingle('vaccinationDate', evt.target.value)}
-                  />
-                  <svg className="omrs-icon" role="img">
-                    <use xlinkHref="#omrs-icon-calendar"></use>
-                  </svg>
-                </div>
-              </div>
-              <div className={styles.immunizationsInputContainer}>
-                <label htmlFor="vaccinationExpiration">{t('expirationDate', 'Expiration Date')}</label>
-                <div className="omrs-datepicker">
-                  <input
-                    type="date"
-                    id="vaccinationExpiration"
-                    name="vaccinationExpiration"
-                    defaultValue={formState.expirationDate}
-                    onChange={(evt) => updateSingle('expirationDate', evt.target.value)}
-                  />
-                  <svg className="omrs-icon" role="img">
-                    <use xlinkHref="#omrs-icon-calendar"></use>
-                  </svg>
-                </div>
-              </div>
-              <div className={styles.immunizationsInputContainer}>
-                <label htmlFor="lotNumber">{t('lotNumber', 'Lot Number')}</label>
-                <div className="omrs-input-group">
-                  <input
-                    className="omrs-input-outlined"
-                    type="text"
-                    id="lotNumber"
-                    style={{ height: '2.75rem' }}
-                    defaultValue={formState.lotNumber}
-                    onChange={(evt) => updateSingle('lotNumber', evt.target.value)}
-                  />
-                </div>
-              </div>
-              <div className={styles.immunizationsInputContainer}>
-                <label htmlFor="manufacturer">{t('manufacturer', 'Manufacturer')}</label>
-                <div className="omrs-input-group">
-                  <input
-                    className="omrs-input-outlined"
-                    type="text"
-                    id="manufacturer"
-                    style={{ height: '2.75rem' }}
-                    defaultValue={formState.manufacturer}
-                    onChange={(evt) => updateSingle('manufacturer', evt.target.value)}
-                  />
-                </div>
-              </div>
-            </div>
+      <Form onSubmit={handleFormSubmit} data-testid="immunization-form">
+        <h4 className={styles.immunizationSequenceSelect}>Vaccine: {formState?.vaccineName} </h4>
```

Comment:
we should translation for this text 
```suggestion
        <h4 className={styles.immunizationSequenceSelect}>{`${t('vaccine','Vaccine')} : ${formState?.vaccineName}`} </h4>
```